java/src/main/java/com/google/oak/remote_attestation/AmdAttestationVerifier.java

```diff
@@ -0,0 +1,46 @@
+//
+// Copyright 2023 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package com.google.oak.remote_attestation;
+
+import com.google.oak.util.Result;
+import oak.session.noninteractive.v1.AttestationEndorsement;
+import oak.session.noninteractive.v1.AttestationEvidence;
+
+/**
+ * Attestation verifier implementation for AMD SEV-SNP.
+ * <https://www.amd.com/system/files/TechDocs/SEV-SNP-strengthening-vm-isolation-with-integrity-protection-and-more.pdf>
+ */
+public class AmdAttestationVerifier implements AttestationVerifier {
+  /**
+   * Verify that the provided evidence was endorsed and contains specified reference values.
+   *
+   * @param evidence contains claims about the Trusted Execution Environment
+   * <https://www.rfc-editor.org/rfc/rfc9334.html#name-evidence>
```

Comment:
This seems to be copied from the javadoc for AttestationVerifier. If that is the case, I think it is better to say that it is copied.

Also, is not it better to customize this according to the AMD-specific logic? (or perhaps leave a TODO to update it as part of #3641).

Reply:
I know. It is common to updated the implementation and forget to update the documentation.

Reply:
Added a new todo
```
// TODO(#3641): Rewrite java-doc to represent actual AMD attestation verification.
```